api/src/main/kotlin/com/github/benmanes/gradle/versions/updates/gradle/GradleUpdateChecker.kt

```diff
@@ -20,7 +20,7 @@ class GradleUpdateChecker(
    * @return if the check for Gradle updates was enabled and, if so, the versions were fetched.
    */
   @get:JvmName("isEnabled")
-  val enabled: Boolean = true,
+  val enabled: Boolean,
```

Comment:
this changes the default to false?

Reply:
This originally had no value - https://github.com/ben-manes/gradle-versions-plugin/blob/c2bad3d8661e203410c80afddcbb3ade4fdc772e/src/main/groovy/com/github/benmanes/gradle/versions/updates/gradle/GradleUpdateChecker.groovy#L26.

I can keep it `true`.

Reply:
https://github.com/ben-manes/gradle-versions-plugin/blob/9f0260e419dc43be340bc048890eef1aaabfd71a/src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdates.groovy#L49